--- a/node.gyp
+++ b/node.gyp
@@ -575,6 +575,8 @@
       ],
 
       'sources': [
+        'src/node_embedding_api.cc',
+        'src/node_embedding_api.h',
         'src/api/async_resource.cc',
         'src/api/callback.cc',
         'src/api/embed_helpers.cc',
--- a/tools/install.py
+++ b/tools/install.py
@@ -123,6 +123,8 @@ def corepack_files(action):
 def subdir_files(path, dest, action):
   ret = {}
   for dirpath, dirnames, filenames in os.walk(path):
+    if sys.platform == 'win32':
+        dirpath = dirpath.replace('\\', '/')
     files_in_path = [dirpath + '/' + f for f in filenames if f.endswith('.h')]
     ret[dest + dirpath.replace(path, '')] = files_in_path
   for subdir, files_in_path in ret.items():
@@ -191,6 +193,7 @@ def headers(action):
     'config.gypi',
     'src/node.h',
     'src/node_api.h',
+    'src/node_embedding_api.h',
     'src/js_native_api.h',
     'src/js_native_api_types.h',
     'src/node_api_types.h',
